### PR TITLE
Revert "[main] Update dependencies from dotnet/arcade (#6409)"

### PR DESCRIPTION
**Pull request by Youssef1313**
This reverts commit adb9c1cf0416e1dc2b547b7a962a171e54cfb85d.

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "8.0.100-alpha.1.23061.8",
+    "dotnet": "7.0.101",
```
Comment (mavasani) [inline global.json:3]: Do we need to revert any of the arcade changes? Can't we just have the two changes in this file that lock tools and sdk dotnet version to "7.0.101" and revert everything else in the PR?

Comment (mavasani) [reply global.json:3]: Maybe the suggestion above might not work. I think someone from the arcade infra team should take a look.

Comment (Youssef1313) [reply global.json:3]: Yeah I tried to only update global.json and that didn't work.

```
.packages/microsoft.dotnet.arcade.sdk/8.0.0-beta.23066.1/tools/SourceBuild/SourceBuildArcadeTools.targets(20,5): error MSB4018: (NETCORE_ENGINEERING_TELEMETRY=Restore) The "ReadSourceBuildIntermediateNupkgDependencies" task failed unexpectedly.
System.IO.FileNotFoundException: Could not load file or assembly 'System.Runtime, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.

```

**Comment by Youssef1313** [comment]
@mavasani Is this ready to merge?

**Comment by mavasani** [comment]
> @mavasani Is this ready to merge? or needs a look from someone from Arcade team?

I would prefer someone from the arcade team to take a look. My primary concern is if this revert locks us to a specific version of arcade and we can't move forward for quite some time. Additionally, we also need to block the auto-generated PRs that keep bumping back the arcade dependency to newer versions. Are you aware which github handle to tag arcade team?

**Comment by Youssef1313** [comment]
@mavasani Not sure, maybe `@dotnet/dnceng` or `@dotnet/arcade-contrib`?

**Comment by MilenaHristova** [comment]
> > @mavasani
> >             
> >             Manish Vasani
> >             FTE Is this ready to merge? or needs a look from someone from Arcade team?
> 
> I would prefer someone from the arcade team to take a look. My primary concern is if this revert locks us to a specific version of arcade and we can't move forward for quite some time. Additionally, we also need to block the auto-generated PRs that keep bumping back the arcade dependency to newer versions. Are you aware which github handle to tag arcade team?

You're right, you can't update to new versions of arcade it is being built with .NET 8 . You should fix the problems in your repo first to make it buildable with the new version

cc: @MichaelSimons

**Comment by mavasani** [comment]
> There is no particular problem. The thought was that it's too early to use .NET 8.
> 
> @mavasani I think we'll have to be on .NET 8? Should we close this PR?

Seems like it

**Pull request closed without merging**